common/libzkp/Makefile

```
@@ -0,0 +1,17 @@
+.PHONY: help fmt clippy test test-ci test-all
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)